.github/workflows/benchmark.yml

```
@@ -6,13 +6,36 @@ jobs:
     name: benchmarking
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@v2
-    - name: Go benchmark
-      uses: bobheadxi/gobenchdata@master
-      with:
-        BENCHMARKS_OUT: .github/benchmarks.json
-        GIT_COMMIT_MESSAGE: 'chore: add new benchmark'
-        PRUNE_COUNT: 30
-        GO_TEST_FLAGS: -cpu 1,2
-      env:
-        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      - 
+        uses: actions/checkout@v2.3.0
+      - 
+        id: vars
+        run: |
+          echo ::set-output name=go_version::$(grep '^FROM go' .github/go/Dockerfile | cut -d ' ' -f 2 | cut -d ':' -f 2)
+          echo "Using Go version ${{ steps.vars.outputs.go_version }}"
+      - 
+        name: Setup go
+        uses: actions/setup-go@v2
+        with:
+          go-version: ${{ steps.vars.outputs.go_version }}
+      - 
+        name: Download Go modules
+        run: go mod download
+      - 
+        name: Install gobenchdata
+        run: |
+          go get go.bobheadxi.dev/gobenchdata
+          git clone https://github.com/bobheadxi/gobenchdata
```

`go get` also does a `git clone` - you can avoid double-cloning by installing from the copy:

```suggestion
          git clone https://github.com/bobheadxi/gobenchdata
          pushd gobenchdata
          go install ./...
          popd gobenchdata
```